crates/blockchain/vm.rs

```diff
@@ -44,16 +44,32 @@ impl StoreVmDatabase {
 impl VmDatabase for StoreVmDatabase {
     #[instrument(level = "trace", name = "Account read", skip_all)]
     fn get_account_info(&self, address: Address) -> Result<Option<AccountInfo>, EvmError> {
-        self.store
+        let result = self
+            .store
             .get_account_info_by_hash(self.block_hash, address)
-            .map_err(|e| EvmError::DB(e.to_string()))
+            .map_err(|e| EvmError::DB(e.to_string()));
+
+        #[cfg(feature = "replay")]
+        if result.is_err() {
+            return Ok(None);
+        };
```

Comment:
We need to check this with @mpaulucci.

Reply:
I think it'd be better to define a more specific error for this case, capture it on the `ethrex-replay` side, and ignore it.

Reply:
@mpaulucci suggested us to implement `VmDatabase` for a `StatelessVmDatabase` struct. I think that it is worth doing since we cannot afford having this feature flag.